Samples/java/src/main/java/com/phunware/java/sample/CustomMapFragment.java

```diff
@@ -0,0 +1,46 @@
+package com.phunware.java.sample;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.util.TypedValue;
+import com.google.android.gms.maps.GoogleMapOptions;
+import com.google.android.gms.maps.model.CameraPosition;
+import com.google.android.gms.maps.model.CameraPosition.Builder;
+import com.google.android.gms.maps.model.LatLng;
+import com.phunware.mapping.MapFragment;
+import com.phunware.mapping.OnPhunwareMapReadyCallback;
+
+/**
+ * Example of extending {@link MapFragment} to customise
+ * the initial camera location and the zooming level of the map.
+ */
+public class CustomMapFragment extends MapFragment {
+
+    public static MapFragment getInstance(AppCompatActivity activity, OnPhunwareMapReadyCallback callback) {
```

Comment:
Should this return `CustomMapFragment` like the kotlin sample?

Reply:
We can do that.
 I'm ok with merging this to sample_code_updates. 
This is probably based off of master. Now that we're having all the code in sample_code_updates, it'll makes sense to add it there